src/pages/RFOX/components/RewardsAndClaims/Claims.tsx

```diff
@@ -1,50 +1,84 @@
+  stakingAssetId: AssetId
+  stakingAssetAccountId: AccountId | undefined
 }
 
-export const Claims = ({ headerComponent }: ClaimsProps) => {
+export const Claims = ({ headerComponent, stakingAssetId, stakingAssetAccountId }: ClaimsProps) => {
   const translate = useTranslate()
   const setConfirmedQuote = useCallback(() => {}, [])
 
-  // @TODO: to be removed when wiring it up
-  const staticKey = '1'
+  const stakingAsset = useAppSelector(state => selectAssetById(state, stakingAssetId))
 
-  return (
-    <CardBody>
-      {headerComponent}
+  const stakingAssetAccountAddress = useMemo(
+    () => (stakingAssetAccountId ? fromAccountId(stakingAssetAccountId).account : undefined),
+    [stakingAssetAccountId],
+  )
 
-      <Box>
-        <ClaimRow
-          stakingAssetId={foxAssetId}
-          key={staticKey}
-          amountCryptoPrecision={'1500'}
-          status={ClaimStatus.CoolingDown}
-          setConfirmedQuote={setConfirmedQuote}
-          cooldownPeriodHuman={'in 1 day'}
-          index={1}
-          actionDescription={translate('RFOX.unstakeFrom', { assetSymbol: 'FOX' })}
-          displayClaimButton
-        />
+  const { data: unstakingRequestResponse, isLoading: isUnstakingRequestLoading } =
+    useGetUnstakingRequestQuery({ stakingAssetAccountAddress })
+
+  const claims = useMemo(() => {
+    if (!stakingAsset) return null
+
+    if (isUnstakingRequestLoading)
+      return new Array(2).fill(null).map(() => <Skeleton height={16} />)
+
+    return (unstakingRequestResponse ?? []).map((unstakingRequest, index) => {
+      const amountCryptoPrecision = fromBaseUnit(
+        unstakingRequest.unstakingBalance.toString(),
+        stakingAsset?.precision ?? 0,
+      )
+      const currentTimestampMs: number = Date.now()
+      const unstakingTimestampMs: number = Number(unstakingRequest.cooldownExpiry) * 1000
+      const isAvailable = currentTimestampMs >= unstakingTimestampMs
+      const cooldownDeltaMs = unstakingTimestampMs - currentTimestampMs
+      const cooldownPeriodHuman = dayjs(Date.now() + cooldownDeltaMs).fromNow()
+      const status = isAvailable ? ClaimStatus.Available : ClaimStatus.CoolingDown
+      return (
         <ClaimRow
-          stakingAssetId={foxAssetId}
-          key={staticKey}
-          amountCryptoPrecision={'1500'}
-          status={ClaimStatus.Available}
+          stakingAssetId={stakingAssetId}
+          key={unstakingRequest.cooldownExpiry.toString()}
+          amountCryptoPrecision={amountCryptoPrecision?.toString() ?? ''}
+          status={status}
           setConfirmedQuote={setConfirmedQuote}
-          cooldownPeriodHuman={'1 day ago'}
-          index={1}
-          actionDescription={translate('RFOX.unstakeFrom', { assetSymbol: 'FOX' })}
-          displayClaimButton
+          cooldownPeriodHuman={cooldownPeriodHuman}
+          index={index}
+          actionDescription={translate('RFOX.unstakeFrom', {
+            assetSymbol: stakingAsset.symbol,
+          })}
```

Comment:
@gomesalexandre can you try to click on any rows?

I think we want to disable this behavior as well as the hover effect as a quick win

Reply:
Tackled in https://github.com/shapeshift/web/commit/1b250fc48180834caa65c41546d9af94f5ce4f3d!